components/script/dom/range.rs

```diff
@@ -362,7 +362,7 @@ impl RangeMethods for Range {
         } else {
             // Step 3.
             self.set_end(node, offset);
-            if !(self.end() >= self.start()) {
+            if self.end() < self.start() {
```

Comment:
Same with this one.

Reply:
It appears to have been reverted already sir.